src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -428,6 +453,28 @@ public ResponseEntity<byte[]> getAttachmentUnitAttachment(@PathVariable Long att
         return buildFileResponse(getActualPathFromPublicPathString(attachment.getLink()), false);
     }
 
+    /**
+     * GET files/courses/{courseId}/attachments/{attachmenUnitId}/file : Returns the file associated with the
+     * given attachmentUnit ID as a downloadable resource
+     *
+     * @param attachmentUnitId the ID of the attachment to retrieve
+     * @return ResponseEntity containing the file as a resource
+     */
+    @GetMapping("files/courses/{courseId}/attachments/attachment-units/{attachmentUnitId}")
+    @EnforceAtLeastEditorInCourse
+    public ResponseEntity<byte[]> getAttachmentUnitFile(@PathVariable Long courseId, @PathVariable Long attachmentUnitId) {
```

Comment:
(server-style) we need a `@param` in the Javadoc for courseId

Reply:
Added